greenhouse/diskutil/diskutil.go

```diff
@@ -70,16 +70,17 @@ func Remount(device, mountPoint, options string) error {
 	return err
 }
 
-// GetDiskUsage wraps syscall.Statfs
-func GetDiskUsage(path string) (percentBlocksFree, percentFilesFree float64, err error) {
+// GetDiskUsage wraps syscall.Statfs for usage in GCing the disk
+func GetDiskUsage(path string) (percentBlocksFree, bytesFree, bytesUsed float64, err error) {
```

Comment:
why is bytesUsed a float? that seems wrong.

Reply:
I was thinking about creating a prometheus guage which is float64, I can change these types and cast later.

Reply:
done